src/Tickets/Commerce/Flag_Actions/Increase_Sales.php

```diff
@@ -65,7 +66,11 @@ public function handle( Status_Interface $new_status, $old_status, \WP_Post $pos
 
 			$global_stock = new \Tribe__Tickets__Global_Stock( $ticket->get_event_id() );
 
-			tribe( Ticket::class )->increase_ticket_sales_by( $ticket->ID, $quantity, $ticket->global_stock_mode(), $global_stock );
+			// Is ticket shared capacity?
+			$global_stock_mode  = $ticket->global_stock_mode();
```

Comment:
Any reason we didn't use https://github.com/the-events-calendar/event-tickets/blob/master/src/Tribe/Tickets_Handler.php#L1000

Reply:
I guess since we have the ticket object, this might make more sense. I'm on the fence.

Reply:
@Camwyn I didn't know about that (TIL), but it looks like `Tickets_Handler::gas_shared_capacity()` is wanting a `WP_Post` object to work with, whereas we have a `Tribe__Tickets__Ticket_Object` as `$ticket` here.

To me it feels like it's easier to use the method here rather than the `Tickets_Handler` class, but with that said I think it would be all around better to have this kind of functionality as part of the ticket object itself in some way.

I don't know enough about the existing code structure to say what that should look like at this point, but maybe the current approach (this PR) is fine for now.

What do you think since you have more experience here?

Reply:
See, that's why I am on the fence - we have the object. I agree it seems like that functionality should move, but that's OOS for this PR/ticket.